app/models/accounting/quickbooks/full_fetcher.rb

```diff
@@ -12,23 +12,35 @@ def initialize(division)
 
       def fetch_all
         accounts = clear_accounts!
+        delete_qb_data
+        fetch_qb_data
+        restore_accounts!(accounts)
+      rescue StandardError => error
+        @qb_connection.destroy
+        ExceptionNotifier.notify_exception(error, data: {})
```

Comment:
I don't think this exception notifier is necessary, it should be caught by the one in either the job or the application controller.

Reply:
@DevneyHamilton Do you want to address this before the ticket is merged?

Reply:
yes, will update. thanks!

Reply:
updated